ui/js/controllers/main.js

```diff
@@ -204,6 +204,21 @@ treeherderApp.controller('MainCtrl', [
          * This new solution uses a simple model scope object and update function
          * to keep things in sync.
          */
+
+        var checkIfSingleTierSelected = function() {
+            var count = 0;
```

Comment:
The logic here could be replaced with this line:

    return _.without(_.values($scope.tiers), false).length == 1

Then if you make this function a ``$scope`` instead of a ``var`` you can just call it from the html.